daft-parquet/query.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+import daft
+import timeit
+import traceback
+import pandas as pd
+from daft import col, DataType, TimeUnit
+from pathlib import Path
+
+hits = None
+
+with open("queries.sql") as f:
+    sql_list = [q.strip() for q in f.read().split(';') if q.strip()]
+
+def daft_offset(df, start ,end):
+    pandas_df = df.to_pandas()
+    sliced_df = pandas_df.iloc[start:end]
+    return sliced_df
+
+queries = []
+for idx, sql in enumerate(sql_list):
+    query_entry = {"sql": sql}
+
+    if idx+1 in [19, 36, 43]:
```

Comment:
Please add comments what the magic constants in l. 24, 25, 34, 26, 47 etc. mean

Reply:
Done